packages/scripts/README.md

```diff
@@ -0,0 +1,125 @@
+
+### Bump
+
+Providing the version, you can update the first `unreleased` literal. `release-it` provides it as a `version` variable.
+
+```shell
+yarn changelogupdater bump ${version}
+```
+
+### Excerpt
+
+It generates the changelog of the version to be released. This is useful when creating the Release information and sending it to GitHub.
+
+```shell
+yarn changelogupdater.js excerpt
+```
+
+### Back
+
+It creates the blank placeholder for the next release. Provided the (next) version, it will be added to the placeholder.
+
+```shell
+yarn changelogupdater.js back ${version}
+```
+
+## addon
+
+This script creates and configure a vanilla project using the Volto generator.
+The project is configured to have the current add-on installed and ready to work with.
+This script is useful to bootstrap an isolated environment that can be used to quickly develop the add-on or for demo purposes.
+It's also useful when testing an add-on in a CI environment.
+
+```{note}
+It's quite similar when you develop a Plone backend add-on in the Python side, and embed a ready to use Plone build (using buildout or pip) in order to develop and test the package.
+```
+
+Unfortunatelly, the NodeJS tooling does not work well with symlinks or outside the root of the project.
+This script proceeds in the following way:
+
+1. Generates a vanilla project using the official Volto Yo Generator (@plone/generator-volto)
+2. Configures is to use the add-on with the name stated in the `package.json`
+3. Copies over the root of the add-on (`src` and essential files) inside the created project
+
+After this, you can change directory to the created project, by default `addon-testing-project`.
+The name of the created project is parameterizable.
+
+After that you can use the full fledged project, and run any standard Volto command for linting, acceptance test or unit tests.
+
+### clone (git)
+
+Given the add-on remote git repository it pulls and configures it into the vanilla project generated by the script.
+
+`npx -p @plone/scripts addon clone [options] <source> [destination]`
+
+    Usage: addon clone [options] <source> [destination]
+
+    clone a repository into a newly created directory
+
+    Options:
+      -p, --private          set if the repo is private, then GITHUB_TOKEN is used
+      -b, --branch <branch>  set the repo branch, defaults to main
+      -c, --canary           downloads latest Volto canary (alpha) version
+      -h, --help             display help for command
+
+This next command, would download the `volto-blocks-grid` add-on from its git repo, branch `main` and will generate a project with the latest Volto canary (alpha) version.
+
+```shell
+npx -p @plone/scripts addon clone https://github.com/kitconcept/volto-blocks-grid.git --branch main --canary
+```
+
+This will create a directory named `addon-testing-project` and will bootstrap a new project using Volto's standard project generator.
```

Comment:
```suggestion
This will create a directory named `addon-testing-project`, and will bootstrap a new project using Volto's standard project generator.
```